docusaurus/docs/iOS/guides/push-notifications.md

```diff
@@ -1,3 +1,179 @@
 ---
 title: Push Notifications
----
\ No newline at end of file
+---
+
+# Setting up basic push notifications
+First step on a way to setting up push notifications is authentication.
+Stream supports both **Certificate-based provider connection trust (.p12 certificate)** and **Token-based provider connection trust (JWT)**. Token based authentication is the preferred way to setup push notifications. This method is easy to setup and provides strong security.
+
+## Setting up token based authentication <br/> Step 1. Retrieve Your Team ID
```

Comment:
All of those steps are explained nicely in CMS: https://getstream.io/chat/docs/php/push_ios/?language=swift

Linking to a common site is helpful since flutter/RN will also link to them, we don't need to replicate content

Reply:
Ok, removed the common part, although as you mentioned it makes sense to push these changes back to CMS because it's slightly outdated